.github/ISSUE_TEMPLATE/release-a-new-version.md

```diff
@@ -26,6 +26,7 @@ assignees: ''
 - [ ] If necessary, update the badges versions in [docs/00 - Stellar Anchor Platform.md].
 - [ ] Create the pull request `release/2.2.x -> main`: this should require two approvals. DO NOT squash merge.
 - [ ] Create another pull request `release/2.2.x -> develop`: AFTER the release branch is merged with the `main` branch. DO NOT squash merge.
+- [ ] Generate the new Anchor Platform documentation version in the stellar-docs repository.
```

Comment:
Do we have to release doc every version? Maybe skip the patch version?

Reply:
Good call. I'll update this.

Reply:
I also remove some steps that we don't do today.